otterdog/adoptium.jsonnet

```diff
@@ -734,6 +734,15 @@ orgs.newOrg('adoptium') {
         orgs.newBranchProtectionRule('main'),
       ],
     },
+    orgs.newRepo('devkit-binaries') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      dependabot_security_updates_enabled: true,
+      description: "Repository for Adoptium OpenJDK DevKit builds",
+      has_issues: false,
```

Comment:
> I'm guessing you do need issues for this?

Not sure, this repository is going to be akin to the temurinNN-binaries, ie.it will hold DevKit tarball "releases". I don't foresee and code or issues in this reprository